### DISCUSSION TITLE
HA use-case for Zinc

### DISCUSSION THREAD

**Discussion by siddharthlatest** [zinclabs/zinc#143] [Ideas]
We provide an API gateway for Elasticsearch with appbase.io. Given Zinc's low mem footprint, it's very attractive as an alternative (perhaps even the default) search engine we support.

Would love to know how your HA timeline looks like. :-)

**Comment by prabhatsharma** [comment]
We still have 3-6 months away for HA.

**Reply by siddharthlatest** [reply]
Just having a public timeline is great, appreciate it! Is there a place where this is publicly tracked?

**Comment by prabhatsharma** [comment]
Not yet

**Comment by prabhatsharma** [comment]
Agree. Will get it up soon.
